orangecontrib/text/widgets/owtwitter.py

```diff
@@ -296,6 +298,7 @@ def update_api(self, key):
 
     @QtCore.pyqtSlot()
     @require('api', MISSED_KEY, MISSED_KEY_MESSAGE)
+    @require('word_list', MISSED_QUERY, MISSED_QUERY_MESSAGE)
```

Comment:
It's supposed to work, but I have got 403 too. But it works with empty authors ('from: ').
So, I will make a PR that fixes that.